scripts/us_census/pep/population_estimate_by_race/state_to_geoid.py

```diff
@@ -0,0 +1,60 @@
+USSTATE_MAP = {
```

Comment:
please use https://github.com/datacommonsorg/data/blob/master/util/alpha2_to_dcid.py and do not duplicate it

Reply:
Hi, this part has been moved as a part of another PR. PEP by Race and Monthly Population has been seperated.